tests/test_annotation.yml

```diff
@@ -5,20 +5,34 @@
     - snpeff
   files:
     - path: results/annotation/test/test_snpEff.ann.vcf.gz
+      md5sum: 01f24fdd76f73eefd695beea7b3d3d8e
     - path: results/annotation/test/test_snpEff.ann.vcf.gz.tbi
+      md5sum: 51e418d9be9bb33f1d4123493b15b6c9
     - path: results/multiqc
     - path: results/reports/snpeff/test/snpEff_summary.html
+    # snpEff_summary.html changes md5sums on reruns.
+    # It should be possible to test snpEff_summary.html using the contains-feature.
+    # (I can't access the output-file on GitHub-test-server and I can't run the tests locally as I can't pull the snpeff- or VEP-image)
```

Comment:
I'll give it another try. I think that I meanwhile solved the problem I had with the snpeff-image on my server.